cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,33 @@
+/// <reference types="cypress" />
+
+const { generateUser } = require('../support/generate');
+
 describe('Sign Up page', () => {
   it('should provide an ability to register new account', () => {
+    const user = generateUser();
+    
+    cy.visit('https://react-redux.realworld.io');
```

Comment:
You could add register endpoint here to start testing from register page without making some extra steps